src/coffea/analysis_tools.py

```diff
@@ -439,11 +945,24 @@ def __init__(self, dtype="uint32"):
         self._names = []
         self._data = None
 
+    def __repr__(self):
+        delayed_mode = None if self._data is None else self.delayed_mode
+        return f"PackedSelection(selections={tuple(self._names)}, delayed_mode={delayed_mode}, items={len(self._names)}, maxitems={self.maxitems})"
+
     @property
     def names(self):
         """Current list of mask names available"""
         return self._names
 
+    @property
+    def delayed_mode(self):
+        if isinstance(self._data, dask_awkward.Array):
+            return True
+        elif isinstance(self._data, numpy.ndarray):
+            return False
+        else:
+            return "PackedSelection hasn't been initialized with a boolean array yet!"
```

Comment:
This function has a problem due to the multiple return value types. 
```python3
if "hello":
    print("world")
```
will output "world", which could lead to unexpected behavior for someone running eager code.

This should likely raise an exception or print a warning and return `False` since unintialized is technically also not delayed mode (but also not eager).